db/migrations/0001_initial.py

```diff
@@ -1,6 +1,7 @@
 # Generated by Django 4.0.2 on 2022-08-19 07:39
```

Comment:
Please do not update existing migtarion. You should represent all changes done to models in one migration file per app per PR.

Reply:
not resolved